tests/gui/test_suggestions.py

```diff
@@ -41,6 +45,13 @@ def test_frame_increment(centered_pair_predictions: Labels):
     )
     assert len(suggestions) == 20
 
+    """ 
+    Testing videos that have less frames than desired Samples per Video (random)
+    
+    Expected result is there should be n suggestions where n is equal to the frames
+    in the video.
+    """
+
```

Comment:
Use docstrings for [function documentation](https://google.github.io/styleguide/pyguide.html#:~:text=A%20docstring%20is%20a%20string%20that%20is%20the%20first%20statement%20in%20a%20package%2C%20module%2C%20class%20or%20function.) only. Use [block comments](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) (just append a # in front of each line of code) here.